operations/observability/mixins/meta/rules/server.yaml

```diff
@@ -47,7 +47,7 @@ spec:
     # Rollout alerts
     - alert: JsonRpcApiErrorRates
       # Reasoning: the values are taken from past data
-      expr: sum(rate(gitpod_server_api_calls_total{statusCode!~"2..|429"}[5m])) by (cluster) / sum(rate(gitpod_server_api_calls_total[5m])) by (cluster) > 0.04
+      expr: sum(rate(gitpod_server_api_calls_total{statusCode!~"2.*|3.*|4.*"}[5m])) by (cluster) / sum(rate(gitpod_server_api_calls_total[5m])) by (cluster) > 0.04
```

Comment:
Why not use the dot syntax, e.g.? `statusCode!~"2..|3..|4.."`

Reply:
habit, I guess. I find the .* a bit more readable, rather than `2..|3..|4..` but it's the same